modules/installation/pages/upgrade.adoc

```diff
@@ -92,42 +92,74 @@ $ gadmin restart all -y
 [#_rolling_upgrade]
 == Rolling Upgrade (Preview)
 
-A rolling upgrade allows you to upgrade your TigerGraph installation with minimal downtime by upgrading nodes in batches, ensuring that at least one full replica of the cluster is online throughout the process. 
+A rolling upgrade allows you to upgrade your TigerGraph installation with minimal downtime by upgrading nodes in batches, ensuring that at least one full replica of the cluster is online throughout the process.
 
-[NOTE]
-====
-Rolling Upgrade is only available for HA clusters (High Availability) and when making a maintenance update (e.g., 4.2.0 to 4.2.1, 4.3.1 to 4.3.3). 
-Upgrading from 4.2.0 to 4.3.0 or any other major version is not supported.
-====
+*Before You Begin*
```

Comment:
Why I don't like this title: Because it doesn't fit the style and purpose of this document. This document is part of a **reference manual** which primarily describes WHAT a feature is and secondarily how to use it, vs a **user guide** which primary describes HOW to perform a particular task. User guides are fine, but that's not what this is.
"Before you begin": who is "you" and before you begin what?  In a reference manual this doesn't make sense because we are focusing on what the feature IS (existential, always true), not on a reading performing some particular task.